pkg/reconciler/parallel/controller.go

```diff
@@ -42,12 +46,23 @@ func NewController(
 
 	parallelInformer := parallel.Get(ctx)
 	subscriptionInformer := subscription.Get(ctx)
+	serviceaccountInformer := serviceaccountinformer.Get(ctx)
+
+	var globalResync func(obj interface{})
+	featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"), func(name string, value interface{}) {
+		if globalResync != nil {
+			globalResync(nil)
+		}
+	})
+	featureStore.WatchConfigs(cmw)
 
 	r := &Reconciler{
-		parallelLister:     parallelInformer.Lister(),
-		subscriptionLister: subscriptionInformer.Lister(),
-		dynamicClientSet:   dynamicclient.Get(ctx),
-		eventingClientSet:  eventingclient.Get(ctx),
+		parallelLister:       parallelInformer.Lister(),
+		subscriptionLister:   subscriptionInformer.Lister(),
+		serviceAccountLister: serviceaccountInformer.Lister(),
+		kubeclient:           kubeclient.Get(ctx),
+		dynamicClientSet:     dynamicclient.Get(ctx),
+		eventingClientSet:    eventingclient.Get(ctx),
 	}
 	impl := parallelreconciler.NewImpl(ctx, r)
```

Comment:
```suggestion
	impl := parallelreconciler.NewImpl(ctx, r, func(impl *controller.Impl) controller.Options {
		return controller.Options{
			ConfigStore: featureStore,
		}
	})
```
we need to decorate the context with the feature store, otherwise we don't see updates on the config maps, when we do later ` feature.FromContext(ctx)`. And the `ConfigStore` option is doing exactly this.